pysam/libcbgzf.pyi

```diff
@@ -0,0 +1,40 @@
+import sys
+
+from typing import Optional, Union, Any, NoReturn
+
+if sys.version_info < (3, 8):
+    from typing_extensions import Literal
+else:
+    from typing import Literal
+
+BUFFER_SIZE: int
+
+class BGZFile:
```

Comment:
Does this class implement any of the IO interfaces?

https://github.com/python/typeshed/blob/master/stdlib/io.pyi

https://github.com/python/typeshed/blob/14d7431ff1f554ae1d103b996995bfe3ecec67bd/stdlib/typing.pyi#L486

Reply:
Probably. What do you suggest?

Reply:
Type annotations for the `gzip` module are here: https://github.com/python/typeshed/blob/master/stdlib/gzip.pyi
We could copy this in the future, but I’m currently focusing on annotating the other parts of pysam.